crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -764,6 +764,15 @@ impl From<Color> for StandardMaterial {
     }
 }
 
+impl From<LinearRgba> for StandardMaterial {
```

Comment:
This implementation doesn't look correct to me: the "obvious" conversion sets the `base_color`, not the emissive field. Either way, this shouldn't be needed.